client/src/components/pages/Profile/index.js

```diff
@@ -57,9 +57,6 @@ class Profile extends Component {
               <h1 className="profile__username">
                 {upperCase(userInfo.username)}
               </h1>
```

Comment:
why you are removing the chat btn ?!!!!!!!
Our issue it is why there is chat btn for the login user just